src/defines.h

```diff
@@ -81,3 +81,9 @@
 #define VDMS_ROI_COORD_Y_PROP "VD:y1"
 #define VDMS_ROI_WIDTH_PROP   "VD:width"
 #define VDMS_ROI_HEIGHT_PROP  "VD:height"
+
+// Videos
+
+#define VDMS_VID_TAG           "VD:VID"
```

Comment:
Inconsistent naming

Reply:
not sure what you mean. I see them consistent. Look at images for reference.

Reply:
Haha. they are consistently inconsistent :)  what's with some caps, some small in the text following :

Reply:
I think I see a pattern. The tags are all caps but property keys are small after :